challengeOneWeek3.js

```diff
@@ -12,4 +12,21 @@
     // return numbers;
 
 
-   
\ No newline at end of file
+    function evenAndOddNumbers(num){
+        for(i=0; i<num.length; i++){
+            if(num[i]%2===0){
+                even.push(num[i])
+            }
+            else if(num[i]%2 !=0){
+                odd.push(num[i]);
+           
+        }
+        }
+        
+    return object={
+        even:even.sort(),
+        odd:odd.sort(),
```

Comment:
sort method doesn't work well with numbers in JavaScript out of the box i.e
```js
// sorting the array below
[2,1, 10,11,20,23].sort()

// gives the result below
[1, 10, 11, 2, 20, 23]
```

We can solve that by passing a comparator function to sort.
e.g
```
const isGreaterThan = (a,b)=>a-b;
[2,1, 10,11,20,23].sort( isGreaterThan);

// the result of the above expression is
[1, 2, 10, 11, 20, 23]
```

See [sort method mdn documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) for more info or we can chat about it later